web/massastation/src/pages/Index/Index.tsx

```diff
@@ -75,60 +75,56 @@ export function Index() {
-                  {Intl.t('index.buttons.search')}
-                </div>
-              </Button>
-              <Button
-                variant="secondary"
-                preIcon={<FiCodepen />}
-                customClass="w-96"
-                onClick={() => navigate('/store')}
-              >
-                <div className="flex items-center mas-buttons">
-                  {Intl.t('index.buttons.explore')}
-                </div>
-              </Button>
+        <div className="flex space-x-8 pb-10">
+          <Button
+            preIcon={<FiGlobe />}
+            customClass="w-1/3"
+            onClick={() => navigate('/search')}
+          >
+            <div className="flex items-center mas-buttons">
+              {Intl.t('index.buttons.search')}
             </div>
-            <DashboardStation
-              imagesDark={[
-                <Image1Dark />,
-                <Image2Dark />,
-                <Image3Dark />,
-                <Image4Dark />,
-                <Image5Dark />,
-              ]}
-              imagesLight={[
-                <Image1Light />,
-                <Image2Light />,
-                <Image3Light />,
-                <Image4Light />,
-                <Image5Light />,
-              ]}
-              components={[
-                <PluginWallet
-                  key="wallet"
-                  isActive={pluginWalletIsInstalled}
-                  title="Massawallet"
-                  iconActive={<WalletActive />}
-                  iconInactive={<WalletInactive />}
-                  onClickActive={() =>
-                    navigate('/plugin/massalabs/wallet/web-app/index')
-                  }
-                  onClickInactive={handleInstallPlugin}
-                />,
-              ]}
-            />
-          </div>
+          </Button>
+          <Button
+            variant="secondary"
+            preIcon={<FiCodepen />}
+            customClass="w-1/3"
+            onClick={() => navigate('/store')}
+          >
+            <div className="flex items-center mas-buttons">
+              {Intl.t('index.buttons.explore')}
+            </div>
+          </Button>
         </div>
+        <DashboardStation
+          imagesDark={[
+            <Image1Dark />,
+            <Image2Dark />,
+            <Image3Dark />,
+            <Image4Dark />,
+            <Image5Dark />,
+          ]}
+          imagesLight={[
+            <Image1Light />,
+            <Image2Light />,
+            <Image3Light />,
+            <Image4Light />,
+            <Image5Light />,
+          ]}
+          components={[
+            <PluginWallet
+              key="wallet"
+              isActive={pluginWalletIsInstalled}
+              title="Massawallet"
+              iconActive={<WalletActive />}
+              iconInactive={<WalletInactive />}
+              onClickActive={() =>
+                navigate('/plugin/massalabs/wallet/web-app/index')
```

Comment:
```suggestion
                navigate(routeFor('/plugin/massalabs/wallet/web-app/index'))
```

Does it work with `routeFor` ?

Reply:
not necessary: https://github.com/massalabs/station/pull/811#discussion_r1235289302